requirements.txt

```diff
@@ -2,3 +2,4 @@ numpy>=1.17.2
 torch>=1.3.1
 pyDeprecate==0.3.*
 packaging
+typing_extensions
```

Comment:
```suggestion
```

`typing_extensions` is among the python built-in packages since `python>=3.5` thus there's no need to specify this package in `requirements.txt`.